### PR TITLE
Edit db entities

### PR DESCRIPTION
**Pull request by Rogon1256**
Add avatar to user entity (file_id)
Add thumbnail to video entity (file_id)
Fix typo in Reaction (LIke -> Like)
Fix Reactions

### CODE REVIEW

**Review by miko089 — DISMISSED** [review]

File: db/models/Video.go [hunk 1]
```diff
@@ -22,6 +22,7 @@ type Video struct {
 	User        *User  `json:"user"`
 	Title       string `json:"title"`
 	Description string `json:"description"`
+	Thumbnail   uint64 `json:"thumbnail"`
```
Comment (SemyonHoyrish) [inline db/models/Video.go]: `Thumnail` is also file, so we need to provide file id there. The same way, as you did in User.go

File: db/models/Reaction.go [hunk 2]
```diff
@@ -14,8 +21,9 @@ type Reaction struct {
 	CommentID uint64 `json:"comment_id"`
 	UserID    uint64 `json:"user_id"`
 
-	IsLIke    bool `json:"is_like"`
-	IsDislike bool `json:"is_dislike"`
+	ReactionType  ReactionType `json:"reaction"`
+	// IsLike    bool `json:"is_like"`
+	// IsDislike bool `json:"is_dislike"`
```
Comment (SemyonHoyrish) [inline db/models/Reaction.go]: We are not migrating to ReactionType. So revert these changes.
Also we do not leave some "debug" comments in production

**Review by SemyonHoyrish — CHANGES_REQUESTED** [review]
Fix things mentioned in conversations

**Review by SemyonHoyrish — APPROVED** [review]
Now it looks good
